vital/exceptions/base.cxx

```diff
@@ -64,21 +64,21 @@ ::set_location( std::string const& file, int line )
 
 // ------------------------------------------------------------------
 char const*
-vital_core_base_exception
+vital_exception
 ::what() const noexcept
 {
   return this->m_what.c_str();
 }
 
 
-// ------------------------------------------------------------------
+// ============================================================================
```

Comment:
Why is this separator line different?

Reply:
Okay... I see why this line is different, but I'm not sure I like this style for separating logical blocks. I'd prefer:

```
//BEGIN vital_exception

// ------------------------------------
...vital_exception methods...

//END vital_exception

///////////////////////////////////////

//BEGIN invalid_value

...etc...
```

...as is used in ViViA ([example](https://github.com/Kitware/vivia/blob/master/Libraries/QtVgWidgets/vgMixerItem.cxx#L80)). Although TBH I'm not sure there is enough code here to need any special separation.

The difference in line lengths is also mildly jarring, though I realize that is because this line is correct, while the others are too short. Maybe since you are "here", the rest of the file could be cleaned up?